src/inet/TCPEndPoint.cpp

```diff
@@ -2147,6 +2133,64 @@ void TCPEndPoint::LwIPHandleError(void *arg, err_t lwipErr)
 
 #if WEAVE_SYSTEM_CONFIG_USE_SOCKETS
 
+INET_ERROR TCPEndPoint::BindSrcAddrFromIntf(IPAddressType addrType, InterfaceId intf)
+{
+    INET_ERROR err = INET_NO_ERROR;
+
+    // If we are trying to make a TCP connection over IPv4 and a 'specified target interface',
+    // then we bind the TCPEndPoint to an IPv4 address on that target interface and use that
```

Comment:
IPv4 or IPv6 address

Reply:
It would apply for both actually. So, have reworded the text.